clients/admin-ui/src/features/datastore-connections/add-connection/ConfigureConnector.tsx

```diff
@@ -123,12 +123,6 @@ const ConfigureConnector: React.FC = () => {
   }, [dispatch]);
 
   useEffect(() => {
-    if (step) {
-      setSelectedItem(
-        connector?.options[Number(step.stepId) - connector.options.length]
-      );
-    }
```

Comment:
I really could not figure out the purpose of this line, and it caused a bug when there's only one tab available, so I got rid of it

Reply:
Bravo tracking this down!